functions.php

```diff
@@ -54,31 +54,13 @@ function course_scripts() {
 		// Enqueque theme scripts.
 		wp_enqueue_script( 'course-header', get_template_directory_uri() . '/assets/js/header.js', [], wp_get_theme()->get( 'Version' ), true );
 
-	}
-
-endif;
```

Comment:
It's a bit weird that we're using `endif;` here when we don't use this style anywhere else in Sensei. Perhaps we could switch to regular `if` statements for consistency?

Reply:
Yap, I was thinking about the same thing, maybe we can remove it in a separate PR